xds/src/main/java/io/grpc/xds/internal/rbac/engine/EvaluateArgs.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine;
+
+import io.grpc.Grpc;
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+
+/** The EvaluateArgs class holds evaluate arguments used in CEL Evaluation Engine. */
+public class EvaluateArgs<ReqT, RespT> {
+  private Metadata headers;
+  private ServerCall<ReqT, RespT> call;
```

Comment:
This is a general comment/question that applies to multiple classes in this PR and not just this class. I noticed the use of `ReqT, RespT` as type params which causes all of these classes (`EvaluateArgs` in this example) to be generic types. However these type params always (unless I miss somewhere) are ultimately plumbed down to a `ServerCall<ReqT, RespT>` and serve no other purpose. So in your code itself you never use `ReqT, RespT` to define/reference any generic types. In that case if you just use `ServerCall<?, ?>` do you lose any functionality or even type checks offered by the Java compiler? I suspect the answer is no .

Reply:
That makes sense. I removed those pointless `<ReqT, RespT>`. Thanks for pointing it out!